### PR TITLE
ims_icscf: add missing documents for database params

### PR DESCRIPTION
**Pull request by hyavari**
ims_icscf: add missing documents for database params:
- Adding missing ims_icscf database parameters to admin documents.
- Change the default value for ims_icscf db_url to Kamailio default db_url value.

ims_usrloc_pcscf: Update the module document
- Fixing the db_url param value from int to string in admin document.

#### Pre-Submission Checklist

- [ :white_check_mark: ] Commit message has the format required by CONTRIBUTING guide
- [ :white_check_mark:] Commits are split per component (core, individual modules, libs, utils, ...)
- [ :white_check_mark:] Each component has a single commit (if not, squash them into one commit)
- [ :white_check_mark:] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [ :white_check_mark:] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [ ] PR should be backported to stable branches
- [ :white_check_mark:] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

#### Description

Only an improvement for users to know they need to set database params for ims_icscf module.

### CODE REVIEW

File: src/modules/ims_icscf/ims_icscf_mod.c [hunk 1]
```diff
@@ -71,7 +71,7 @@ stat_var *uar_replies_response_time = NULL;
 stat_var *lir_replies_response_time = NULL;
 
 //module parameters
-char *ims_icscf_db_url = "mysql://icscf:heslo@localhost/hssdata"; /**< DB URL */
+char *ims_icscf_db_url = "mysql://kamailio:kamailiorw@localhost/kamailio"; /**< DB URL */
```
Comment (henningw) [inline src/modules/ims_icscf/ims_icscf_mod.c:74]: Consider using the correct define for it: DEFAULT_RODB_URL

File: src/modules/ims_icscf/doc/ims_icscf_admin.xml [hunk 2]
```diff
@@ -111,6 +111,106 @@ modparam("ims_icscf","scscf_entry_expiry", 300)
...
+        <emphasis>
+            Default value is <quote>&defaultrodb;</quote>.
+        </emphasis>
+      </para>
+
+      <example>
+        <title>Set <varname>db_url</varname> parameter</title>
+        <programlisting format="linespecific">
+...
+modparam("ims_icscf", "db_url", "mysql://username:password@localhost/kamailio")
```
Comment (henningw) [inline src/modules/ims_icscf/doc/ims_icscf_admin.xml:136]: You could use also here the definition "&exampledb;

**Review by henningw — COMMENTED** [review]
Thanks for the PR, I have added two comments related to the usage of database URLs.

File: src/modules/ims_icscf/ims_icscf_mod.c (hunk 1 above)
Comment (hyavari) [reply src/modules/ims_icscf/ims_icscf_mod.c]: Thanks, I resolved it.

File: src/modules/ims_icscf/doc/ims_icscf_admin.xml (hunk 2 above)
Comment (hyavari) [reply src/modules/ims_icscf/doc/ims_icscf_admin.xml]: Thanks, I applied it.

**Pull request merged**